### DISCUSSION TITLE
emit() got multiple values for argument 'to'

### DISCUSSION THREAD

**Discussion by viktorstrand** [miguelgrinberg/python-socketio#728] [Q&A]
I get this error message trying to emit an event from my server: emit() got multiple values for argument 'to', and I do not know how to fix it.
<img width="395" alt="Skärmbild 2021-03-14 230224" src="https://user-images.githubusercontent.com/46597051/111127908-9d7f5380-8574-11eb-9c53-524e344fcbe5.png">

**Comment by miguelgrinberg** [comment]
What are `player_id` and `player_name` supposed to be? Are they both data arguments for the event? If they are, they should be sent as a tuple.
